### ISSUE TITLE
[MAS1.4.3] Luminosity Contrast Ratio of 'Upload File' and 'Send' button icons is less than required.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1305]
**Actual Result:**
Luminosity Contrast Ratio of 'Upload File' button icon is 2.8:1 and 'Send' button icon is 1.7:1.

**Expected Result:**
Luminosity Contrast of 'Upload File' and 'Send' button icons should be greater than or equal to 4.5:1.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](url) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Open 'Colour Contrast Analyser' and select Foreground & Background color of mentioned elements one by one.
4. Notice value of Luminosity Contrast Ratio.
 
**User Impact:**
Users with low vision or Color blindness will find it difficult to access content with low luminosity contrast ratio.

**MAS Reference:**
[MAS 1.4.3 – Contrast (Minimum) (17)](url)

**Test Environment:**
OS: Windows 10
OS Build: 17134.345
OS Version: 1803
Browser: Edge
Tool: Colour Contrast Analyser

**Attachment:**
![mas1 4 3_botframework_webchat_uploadfile_send](https://user-images.githubusercontent.com/41668935/47962387-1cc5f980-e042-11e8-88d0-d84b1d7c98da.png)

**Comment by compulim** [comment]
`#767676` is the minimal gray for 4.5:1. We were using `#999999`.

**Comment by v-rampul** [comment]
#HCL_BotFramework_WebChat

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG1.4.3
